src/main/java/de/tum/cit/aet/artemis/programming/service/ProgrammingAssessmentService.java

```diff
@@ -88,6 +91,10 @@ private Result saveManualAssessment(Result result, User assessor) {
      * @return the new saved result
      */
     public Result saveAndSubmitManualAssessment(StudentParticipation participation, Result newManualResult, Result existingManualResult, User assessor, boolean submit) {
+        // we do this to avoid problems of editing longFeedbackTexts in manual Assesements
+        resultService.deleteLongFeedback(newManualResult.getFeedbacks());
+        List<Feedback> feedbacks = new ArrayList<>(newManualResult.getFeedbacks());
+        newManualResult.updateAllFeedbackItems(feedbacks, true);
```

Comment:
This seems to be a code duplicate with the same 4 lines in `src/main/java/de/tum/cit/aet/artemis/assessment/service/ResultService.java` - could you please generalize it to a method?